content/hi/automation.md

```diff
@@ -0,0 +1,12 @@
+---
+टाइटल: ऑटोमेशन 
+---
+## ऑटोमेशन का क्या मतलब है ?
+ऑटोमेशन का अर्थ स्वचालन हैं । यह एक तकनीक है जिसमे किसी भी मशीन को फुल्ली आटोमेटिक बनाया जाता हैं आसान भाषा में बताया जाए तो ये एक लेबर सेविंग टेक्नोलॉजी हैं जिसमे किसी भी काम को करने के लिए कम से कम लोगो की आवशकता होती हैं, उद्धहरण के लिए जैसे : चैटबॉट , जब भी आप कोई वेबसाइट पर विजिट करते है तो आपके पास एक हेलो पॉपअप होता है तो वो एक ऑटोमेशन है | आजकल ऑटोमेशन हर क्षेत्र में प्रयोग किया जाता है | 

```

Comment:
- फुल्ली should be पूरी तरह से
- ',' should be there after आटोमेटिक बनाया जाता हैं
- लेबर सेविंग can be replaced with `परिश्रम मुक्ति`
- आवशकता is wrong, should be `आवश्यकता`